code/game/objects/structures/stool_bed_chair_nest/stools.dm

```diff
@@ -74,3 +73,15 @@
 		T.Weaken(5)
 		return
 	..()
+
+/obj/item/weapon/stool/dropped(mob/user) //Make sure we aren't leaving un-deconstructible stools.
+	..()
+	if(!user.in_throw_mode) //If this stool isn't getting thrown, turn it back into a structure.
+		origin.forceMove(user.loc)
+		qdel(src)
+
+/obj/item/weapon/stool/throw_impact(atom/hit_atom) //Thrown but uncaught stools will automatically turn into the structure version as soon as the throw is over.
+	..()
+	if(!ishuman(loc)) //If it wasn't caught by someone, put the structure version down.

```

Comment:
Upon further examination, it seems regular old carbons can catch stuff. I'll change this check